fishy/engine/semifisher/fishing_event.py

```diff
@@ -34,11 +34,12 @@ class FishEvent:
     uid = None
     sound = False
 
-def _fishing_sleep(waittime, lower_limit = 16, upper_limit = 58):
+def _fishing_sleep(waittime, lower_limit_ms = 16, upper_limit_ms = 2500):
     reaction = 0.0
-    if (FishEvent.jitter):
-        reaction = float( random.randrange(lower_limit, upper_limit) )/100.0
-    time.sleep(waittime+reaction)
+    if (FishEvent.jitter and (upper_limit_ms > lower_limit_ms)):
```

Comment:
no need for brackets

Reply:
I make them for readability and I like them in this and the other case.

Reply:
It's saying, look this part is a logical unit.

Reply:
actually its pythonic to not use brackets until there are alot of `and` and `or` conditions u need to group up... for a simple `and` between two bools, its better if brackets doesn't clutter up the code

Reply:
Okay If you want it that way. In my opinion having a comparison makes it complicated to see on first glance.

Reply:
You want the outer brackets gone too, right?

Reply:
> In my opinion having a comparison makes it complicated to see on first glance.

I understand, I too used to have the same opinion coming from java to python, but then years of using pycharms it started to feel like brackets clutter up the code for no reason

> You want the outer brackets gone too, right?

Yes, and I think you did it, so accepting the pr